pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -473,7 +471,7 @@ def plot_field(
         ----------
         quantity : str
             Quantity to plot (e.g. ``"Mag_E"``).
-        object_list : str, list
+        objects : str, list
             Objects or faces to which apply the Field Plot.
         plot_type  : str, optional
             Plot type. Options are ``"Surface"``, ``"Volume"``, ``"CutPlane"``.
```

Comment:
```suggestion
            Objects or faces to apply the field plot to.
        plot_type  : str, optional
            Plot type. Options are ``"CutPlane"``, ``"Surface"``, and ``"Volume"``.
```
Alphabetize options.